### PR TITLE
mry-bug-bash

### PR DESCRIPTION
**Pull request by mry-usds**
This PR ... 

## Changes
- 
- 

## Checklist
- [ ] Tested locally?
- [ ] Ran `quickTest all`?
- [ ] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from http://localhost:7071/api/download
- [ ] Updated the release notes? 
- [ ] Added tests?
- [ ] Did you check for sensitive data, and remove any? 
- [ ] Does logging contain sensitive data?  
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Security
*List potential security threats and mitigations in the PR*

## Fixes
*List GitHub issues this PR fixes*
- #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -5,8 +5,6 @@
     "axios": "^0.21.1",
     "chart.js": "^3.1.0",
     "cypress": "^7.0.1",
-    "js-yaml": "^4.0.0",
```
Comment (drew-usds) [inline frontend/package.json:8]: We need to keep YAML for the page content data files

File: frontend/src/_includes/navbar.liquid [hunk 2]
```diff
@@ -40,3 +41,12 @@
     </nav>
   </div>
 </header>
+
+<script>
+  let signIn = document.getElementById("signIn");
+  if( window.sessionStorage.getItem("jwt") )
+    signIn.style.display = "none";
```
Comment (drew-usds) [inline frontend/src/_includes/navbar.liquid:48]: When logged out, this is defaulting to an inline `display: none` for me and not showing the "Sign in" button
 
Screen Shot 2021-04-21 at 2 03 52 AM

**Comment by drew-usds** [comment]
@mry-usds made a couple notes above. Otherwise runs ok on my localhost. I can't see any data locally so can't test other updates until this is up on staging.

File: frontend/package.json (hunk 1 above)
Comment (drew-usds) [reply frontend/package.json:8]: Whoops NM! I see now that this was getting declared twice!

File: frontend/src/_includes/navbar.liquid (hunk 2 above)
Comment (mry-usds) [reply frontend/src/_includes/navbar.liquid:48]: weird - not seeing that; I can log out and the sign in button appears;

**Review by drew-usds — APPROVED** [review]

**Pull request merged**